tool-plugins/nuxt-starter/nuxt.config.ts

```diff
@@ -0,0 +1,30 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+	devtools: { enabled: true },
+	ssr: false,
+	extends: [
+		[
+			'github:storyblok/space-tool-plugins/space-plugins/nuxt-base',
+			{ install: true },
+		],
+		[
+			'github:storyblok/space-tool-plugins/space-plugins/nuxt-base-ui',
```

Comment:
```suggestion
			'github:storyblok/space-tool-plugins/tool-plugins/nuxt-base-ui',
```

Reply:
I actually used the same nuxt-base and nuxt-base-ui from space-plugins folder. I didn't duplicate the same inside tool plugins folder.